Tests/test_Clustalw_tool.py

```diff
@@ -120,9 +120,10 @@ def standard_test_procedure(self, cline):
         output_records = SeqIO.to_dict(SeqIO.parse(cline.outfile, "clustal"))
         self.assertTrue(set(input_records.keys()) == set(output_records.keys()))
         for record in align:
-            self.assertTrue(str(record.seq) == str(output_records[record.id].seq))
-            self.assertTrue(str(record.seq).replace("-", "") ==
-                   str(input_records[record.id].seq))
+            self.assertEqual(str(record.seq),
+                             str(output_records[record.id].seq))
+            self.assertEqual(str(record.seq).replace("-", ""),
```

Comment:
Good fix - I can't use the excuse this was only supported in Python 2.7 onwards, it should probably always have been ``self.assertEqual``